pipelines/build/common/build_base_file.groovy

```diff
@@ -223,15 +223,19 @@ class Builder implements Serializable {
     */
     Boolean getReproducibleCompare(Map<String, ?> configuration, String variant) {
```

Comment:
another thing is about the function name. 
maybe something like `isEnableReproducibleCompare` is better than `getReproducibleCompare` ?